<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WOptionUI</class>
 <widget class="QDialog" name="WOptionUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="Widget2">
      <attribute name="title">
       <string>&amp;Save/Load</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="save_window_p">
         <property name="text">
          <string>S&amp;ave/Restore window position</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="reload_p">
         <property name="text">
          <string>&amp;Reload automatically the last project</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="save_on_exit_p">
         <property name="text">
          <string>Save project automatically on exit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reset_warning_p">
         <property name="text">
          <string>Showl all warnings again</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TabPage">
      <attribute name="title">
       <string>C&amp;omments</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="textLabel2_2">
         <property name="text">
          <string>&amp;Font:</string>
         </property>
         <property name="buddy">
          <cstring>source_code_font_p</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="comment_font_p">
         <property name="text">
          <string>?</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="textLabel1">
         <property name="text">
          <string>&amp;Minimum Comment Size:</string>
         </property>
         <property name="buddy">
          <cstring>minimum_comment_size_p</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="minimum_comment_size_p">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>25</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="no_comment_for_validation_p">
         <property name="text">
          <string>Do not request a comment when setting 
an item to the state &quot;manually validated&quot;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TabWatermark">
      <attribute name="title">
       <string>&amp;Watermarks</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Source File</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="textLabel1_2_2">
            <property name="text">
             <string>&lt;font color=&quot;#ff8000&quot;&gt;Medium&lt;/font&gt;/&lt;font color=&quot;#00ff00&quot;&gt;High&lt;/font&gt; Coverage Level:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="module_medium_level_p">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="textLabel1_2">
            <property name="text">
             <string>&lt;font color=&quot;#ff0000&quot;&gt;Low&lt;/font&gt;/&lt;font color=&quot;#ff8000&quot;&gt;Medium&lt;/font&gt; Coverage Level:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="module_low_level_p">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Namespaces/Class</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="textLabel1_2_3">
            <property name="text">
             <string>&lt;font color=&quot;#ff8000&quot;&gt;Medium&lt;/font&gt;/&lt;font color=&quot;#00ff00&quot;&gt;High&lt;/font&gt; Coverage Level:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="class_medium_level_p">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="textLabel1_5">
            <property name="text">
             <string>&lt;font color=&quot;#ff0000&quot;&gt;Low&lt;/font&gt;/&lt;font color=&quot;#ff8000&quot;&gt;Medium&lt;/font&gt; Coverage Level:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="class_low_level_p">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Method/Function/Procedure</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="textLabel1_2_4">
            <property name="text">
             <string>&lt;font color=&quot;#ff8000&quot;&gt;Medium&lt;/font&gt;/&lt;font color=&quot;#00ff00&quot;&gt;High&lt;/font&gt; Coverage Level:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="function_medium_level_p">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="textLabel1_6">
            <property name="text">
             <string>&lt;font color=&quot;#ff0000&quot;&gt;Low&lt;/font&gt;/&lt;font color=&quot;#ff8000&quot;&gt;Medium&lt;/font&gt; Coverage Level:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="function_low_level_p">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="statistic_background_p">
         <property name="text">
          <string>Calculation of statistics in background</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TabDisplay">
      <attribute name="title">
       <string>&amp;Display</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="textLabel1_3">
         <property name="text">
          <string>&amp;Main Font:</string>
         </property>
         <property name="buddy">
          <cstring>main_font_p</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="main_font_p">
         <property name="text">
          <string>?</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="textLabel2">
         <property name="text">
          <string>So&amp;urce Code Font:</string>
         </property>
         <property name="buddy">
          <cstring>source_code_font_p</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="source_code_font_p">
         <property name="text">
          <string>?</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="textLabel1_4">
         <property name="text">
          <string>Tab S&amp;ize:</string>
         </property>
         <property name="buddy">
          <cstring>tab_size_p</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="tab_size_p">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>16</number>
         </property>
         <property name="value">
          <number>8</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Tool Tip:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <layout class="QHBoxLayout">
         <item>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QCheckBox" name="tooltip_method_list_p">
             <property name="text">
              <string>&amp;Method List</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="tooltip_source_view_p">
             <property name="text">
              <string>Sou&amp;rce Window</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QCheckBox" name="tooltip_execution_list_p">
             <property name="text">
              <string>E&amp;xecution List</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="tooltip_source_list_p">
             <property name="text">
              <string>So&amp;urce List</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="toolbar_p">
         <property name="text">
          <string>&amp;Tool bar settings...</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Window &amp;Style:</string>
         </property>
         <property name="buddy">
          <cstring>window_style_p</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QComboBox" name="window_style_p"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="language_label_p">
         <property name="text">
          <string>&amp;Language:
(Restart is needed)</string>
         </property>
         <property name="buddy">
          <cstring>language_p</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QComboBox" name="language_p"/>
       </item>
       <item row="7" column="2">
        <widget class="QSpinBox" name="executed_by_limit_p">
         <property name="suffix">
          <string> executions</string>
         </property>
         <property name="maximum">
          <number>99999</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Executed By &amp;Limit</string>
         </property>
         <property name="buddy">
          <cstring>executed_by_limit_p</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Cache">
      <attribute name="title">
       <string>Cache</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QLabel" name="execution_cache_label_p">
              <property name="text">
               <string>&amp;Execution Cache</string>
              </property>
              <property name="buddy">
               <cstring>execution_cache_p</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="execution_cache_p">
              <property name="minimum">
               <number>100</number>
              </property>
              <property name="maximum">
               <number>1000000</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="execution_cache_info_p">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QLabel" name="source_cache_label_p">
              <property name="text">
               <string>&amp;Source Cache</string>
              </property>
              <property name="buddy">
               <cstring>source_cache_p</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="source_cache_p">
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>10000</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="source_cache_info_p">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QVBoxLayout" name="_2">
          <item>
           <layout class="QHBoxLayout" name="_3">
            <item>
             <widget class="QLabel" name="statistic_cache_label_p">
              <property name="text">
               <string>Sta&amp;tistic Cache</string>
              </property>
              <property name="buddy">
               <cstring>statistic_cache_p</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="statistic_cache_p">
              <property name="minimum">
               <number>1000</number>
              </property>
              <property name="maximum">
               <number>1000000</number>
              </property>
              <property name="singleStep">
               <number>1000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="statistic_cache_info_p">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>QPixmap</pixmapfunction>
 <resources/>
 <connections/>
</ui>
